recipe/build.sh

```diff
@@ -5,11 +5,32 @@ set -euxo pipefail
 mkdir -p build
 pushd build
 
-if [[ "${target_platform}" == "osx-arm64" ]]; then
-  export CMAKE_ARGS="${CMAKE_ARGS} -DDUCKDB_PLATFORM=osx_arm64 -DDUCKDB_EXPLICIT_PLATFORM=osx_arm64"
+if [[ "${target_platform}" == "linux-64" ]]; then
+    DUCKDB_ARCH='linux_amd64'
+elif [[ "${target_platform}" == "linux-ppc64le" ]]; then
+    DUCKDB_ARCH='linux_amd64'
```

Comment:
This is definitely the wrong `DUCKDB_ARCH`. Not sure where to find the possible values.

Reply:
Let's skip `ppc64le` for now.

Reply:
Please note that the `DUCKDB_ARCH ` env var is only used to specify the directory where the extension will be stored. It's not used during compilation.

Reply:
Ah, then you can invent stuff, right? Then let's do this.

Reply:
> Ah, then you can invent stuff, right? Then let's do this.
Yes. It just needs to match what DuckDB does to determine the platform. DuckDB expects extensions to be stored under `<extension dir>/<duckdb version>/<system architecture>/*`. I don't know what it maps `ppc64le` to. But that should be relatively easy to figure out and unrelated to the error we see when finalizing the extension build.